content/concepts/introduction-to-webpack.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Introduction
+---
+
+*Webpack* is a module system for modern Javascript applications.
+
+In [modular programming](https://en.wikipedia.org/wiki/Modular_programming), developers break programs up into discrete chunks of functionality.
+This has a number of benefits.
+Each module has a smaller surface area for testing than a full program, making verification and bug-hunting easier.
+Well-written modules provide solid abstractions and encapsulation boundaries, so that each module has a coherent design and a clear purpose within the overall application.
+
+Node has supported modular programming almost since its inception.
```

Comment:
I like Node.js. Maybe all these things go into a writing guide. lol another item on the todo list.